shell/probe/raindrop.ps1

```diff
@@ -1,34 +1,36 @@
 function Run {
     Param([String]$Dat = "")
+    try {
```

Comment:
there is a lot of changes going into this file... hard to tell without a side by side, but is this entirely just a try/catch/finally change, to ensure the file is removed in a new finally block?

Reply:
i believe so

Reply:
That's right